src/overview/config.d

```diff
@@ -33,19 +33,29 @@ Configuration readConfigFile (string path)
     import std.exception : ifThrown;
     import dyaml.loader;
     import dyaml.node;
+    import vibe.core.log : logWarn;
 
     auto yml = Loader(path).load();
 
     OctodConfiguration octod;
     octod.dryRun = yml["dry_run"].get!bool.ifThrown(true);
     octod.oauthToken = yml["oauthtoken"].get!string;
 
-    return typeof(return)(
+    auto conf = typeof(return)(
         octod,
         yml["organization"].get!string,
         yml["exclude"].get!(Node[])
             .map!(node => node.get!string).array().ifThrown(null),
         yml["include"].get!(Node[])
             .map!(node => node.get!string).array().ifThrown(null),
     );
+
+    if (conf.octod.dryRun)
+        logWarn("Using dry run mode, no HTTP requests will be sent");
+    if (conf.organization.length == 0)
+        logWarn("Empty organization name, likely misconfiguration");
```

Comment:
Also, maybe it makes sense to make this mandatory, not just a warning. Does it make sense to run without an organization? It will scan the whole github?